assets/js/googlesitekit/data/create-settings-store.test.js

```diff
@@ -0,0 +1,453 @@
+/**
+ * Settings datastore functions tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { createRegistry } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import {
+	muteConsole,
+	subscribeUntil,
+	unsubscribeFromAll,
+} from 'tests/js/utils';
+import { createSettingsStore } from './create-settings-store';
+
+const SETTING = {
+	slug: 'testSetting',
```

Comment:
Rather than calling this `testSetting`, using a more illustrative example might help with readability and understanding of how this works. The word "setting" is already a bit vague and is used a lot throughout this code.

Even something not-serious like `isSkyBlue` or something might be a bit more illustrative. Just a thought! 😄